examples/amp-subscriptions.amp.html

```diff
@@ -207,9 +207,13 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section subscriptions-section="actions" class="login-section">
-          <div subscriptions-action="login" subscriptions-display="NOT loggedIn">Login</div>
-        </section>
+
+        <div subscriptions-action="login" subscriptions-display="NOT loggedIn">Login Local</div>
+
+        <div subscriptions-display="NOT loggedIn"
+          subscriptions-action="login"
+          subscriptions-service="subscribe.google.com"
+          subscriptions-decorate>Login</div>
```

Comment:
One note below in code: let's make this `true` by default. Also, please add docs to md file for both `subscriptions-service` and `subscriptions-decorate`.

Reply:
Done